### PR TITLE
Build for files in moc.wiki moc-public.wiki

### PR DESCRIPTION
**Pull request by hgibson1**
I checked that all the documentation could build and all links were working, but DID not check for the accuracy or up-to-date-ness of the information. Here's what the docs look like built https://documentation-hgibson-fork.readthedocs.io/en/latest/

### CODE REVIEW

File: docs/source/Flash-Drive-Sign-Out.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Flash Drives
```
Comment (naved001) [inline docs/source/Flash-Drive-Sign-Out.md]: we can delete this document,

File: docs/source/Fuel-openstack-install-(on-centos).md [hunk 2]
```diff
@@ -0,0 +1 @@
+Installed CentOS on Computer 10 (cs-jabba).
```
Comment (naved001) [inline docs/source/Fuel-openstack-install-(on-centos).md]: we can probably delete this? it's just one line of pretty much nothing

File: docs/source/Fujitsu-CD10000.md [hunk 3]
```diff
@@ -0,0 +1,568 @@
+# Fujitsu CD10000
+
+### Login Info
+  
+**Web URL** : https://129.10.3.253/dashboard (note - web server no longer responding on this address)
+  
+**User/Password** : "admin"/"90@Limber@Joyriding" (darned password requirements...)
+
+**SSH User/Password** : root/same password as above. The OLD password was 'fjCD#1007', if it's ever relevant. fjCD#1007 is still the root password for nodes 1-4
```
Comment (naved001) [inline docs/source/Fujitsu-CD10000.md]: bunch of passwords here,

File: docs/source/Fujitsu-CD10000.md [hunk 4]
```diff
@@ -0,0 +1,568 @@
...
+
+### IPMI
+
+**IPMI of the management server** : 10.99.1.14 (accessible via HIL master or emergency box)
+
+* current user/password:  admin/3YKHDPQgmMkf1
```
Comment (naved001) [inline docs/source/Fujitsu-CD10000.md]: password

File: docs/source/Kumo-(16-Blade)-(R4-PA-C23)-Documentation.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+     10.10.10.202    Supermicro 2    Services Box
+     10.10.10.203    Dell R720 Storage Server
+     ...
+     10.10.10.210    Dell M1000e CMC
+     10.10.10.{211-226}    IDRAC for Dell Blades 1-16
+  
+### Cisco Nexus 3548 Switch
+    
+    username: admin
+    password: P@ssword1
```
Comment (naved001) [inline docs/source/Kumo-(16-Blade)-(R4-PA-C23)-Documentation.md]: switch password

File: docs/source/Kumo-(16-Blade)-(R4-PA-C23)-Documentation.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+### Cisco Catalyst 3650 Switch
+<!--
+[Catalyst 3650 Hardware Install Guide](manuals/cat3650_hardware_install_guide.pdf)  
+[Catalyst 3650 Getting Started Guide](manuals/cat3650_getting_started_guide.pdf)
+-->
+     username: admin
+     ssh/telnet password: cisco
+     enable password: wittinglypreconfigure
```
Comment (naved001) [inline docs/source/Kumo-(16-Blade)-(R4-PA-C23)-Documentation.md]: passwords

File: docs/source/Configuration-of-the-Helpdesk-VMs.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+**NOTE: This is the helpdesk VM used by UMass helpdesk**
+
+The vm is `moc-helpdesk` in the `moc-infrastructure` project on Kaizen.  Currently Rado and Austin have access to this project, and to the VM.
+
+IP: 128.31.25.252
+
+Console login: 
+
+user: `root`
+password: `BsuYAi1HC53omFFVo+`
```
[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Configuration-of-the-Helpdesk-VMs.md]: Ignore/delete the password. access is with ssh keys, ip is no longer the same we use helpdeskvm.massopen.cloud

File: docs/source/Configuration-of-the-Helpdesk-VMs.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+***
+
+### Engage1 Helpdesk VM
+
+IP: 128.31.22.62
+
+Console login: 
+
+user: `centos`
+password: `KWWA44dIKyEx`
```
[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Configuration-of-the-Helpdesk-VMs.md]: Ignore/delete the password. access is with ssh keys, ip is no longer the same we use helpdeskvm.massopen.cloud

[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Configuration-of-the-Helpdesk-VMs.md]: And we should double check password auth is disabled over ssh

File: docs/source/Death-Star-VM-configuration.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+We're running four VMs on death-star, using virt-manager:  moc-dhcp, moc-nfs, moc-nova and moc-gateway.  All are set to autostart.  All machines used bridged networking, on one or both of the two bridged networks on death-star.
```
[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Death-Star-VM-configuration.md]: No idea about that one

File: docs/source/Engage-1-Bigdata-Environment.md [hunk 10]
```diff
@@ -0,0 +1 @@
+Engage 1 Bigdata Environment
```
[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Engage-1-Bigdata-Environment.md]: Crap

File: docs/source/Engage1-Management-Cable-Map.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
+# Engage1 Management Cable Map
+*Note - default configuration is access on Vlan1*
+
+### egg1-r4pAc04-mgmt  
+* switch management at 10.10.10.1
+* credentials admin/cisco
+
+to change anything, type `enable` and password: `wittinglypreconfigure`
```
[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Engage1-Management-Cable-Map.md]: Put in BW and remove from docs

File: docs/source/Engage1-Management-Cable-Map.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
+# Engage1 Management Cable Map
+*Note - default configuration is access on Vlan1*
+
+### egg1-r4pAc04-mgmt  
+* switch management at 10.10.10.1
+* credentials admin/cisco
```
[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Engage1-Management-Cable-Map.md]: Put in BW and remove from docs

File: docs/source/Engage1-Management-Cable-Map.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+gig 1/0/45 | trunk | 3040, 4004 | trunk linke to Kumo switch | green
+gig 1/0/46 | trunk | 1601, 1602, 3040, 4004 | trunk line to Dell MRI switch | navy blue
+gig 1/0/47 | trunk | 3040, 4004 | trunk line to egg1-r4pAc02-mgmt | red
+gig 1/0/48 | access | 3040 | -- | Local Reserved
+
+
+### egg1-r4pAc04-mgmt-02
+* Juniper ex3300
+* switch management at 10.10.10.5
+* credentials: root/admin38
```
[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Engage1-Management-Cable-Map.md]: Put in BW and remove from docs

File: docs/source/Engage1-Management-Cable-Map.md [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+gig 1/0/22 | access | 3040 | -- | --
+... | ... | ... | ... | 
+gig 1/0/45 | access | 3040 | -- | --
+gig 1/0/46 | access | 3040 | to e1-r4pAc01-mgmt p48 | gray
+gig 1/0/47 | access | 3040 | -- | Local Reserved
+
+
+### egg1-r4pAc02-mgmt  
+* switch management at 10.10.10.2
+* credentials admin/cisco
```
[the remark this answers is not in the document]
Comment (radonm) [reply docs/source/Engage1-Management-Cable-Map.md]: Put in BW and remove from docs

File: docs/source/Flash-Drive-Sign-Out.md (hunk 1 above)
Comment (radonm) [reply docs/source/Flash-Drive-Sign-Out.md]: Delete

File: docs/source/Fuel-openstack-install-(on-centos).md (hunk 2 above)
Comment (radonm) [reply docs/source/Fuel-openstack-install-(on-centos).md]: Seems crap

File: docs/source/Fujitsu-CD10000.md (hunk 3 above)
Comment (radonm) [reply docs/source/Fujitsu-CD10000.md]: Appliance no longer exists - remove

File: docs/source/Fujitsu-CD10000.md (hunk 4 above)
Comment (radonm) [reply docs/source/Fujitsu-CD10000.md]: Delete

File: docs/source/Kumo-(16-Blade)-(R4-PA-C23)-Documentation.md (hunk 5 above)
Comment (radonm) [reply docs/source/Kumo-(16-Blade)-(R4-PA-C23)-Documentation.md]: Put in BW and remove from docs

File: docs/source/Kumo-(16-Blade)-(R4-PA-C23)-Documentation.md (hunk 6 above)
Comment (radonm) [reply docs/source/Kumo-(16-Blade)-(R4-PA-C23)-Documentation.md]: Put in BW and remove from docs

**Pull request merged**